src/ioc-extras.ts

```diff
@@ -53,6 +53,11 @@ export const Execa = Symbol('execa');
  */
 export const FS = Symbol('FS');
 
+/**
+ * Injection for the `FsUtils` module.
+ */
+export const FSUtils = Symbol('FsUtils');
```

Comment:
Since the FS utils are an interface we own, please rename this to `IFSUtils` and place it in the file where that interface is declared. This will merge the declarations, e.g.

https://github.com/microsoft/vscode-js-debug/blob/25d128bf4749da1c301bb62e9cd3a7cd143bc503/src/adapter/performance/index.ts#L12-L19

Reply:
Done